embeddings/bq-vector-search-log-outlier-detection.ipynb

```diff
@@ -0,0 +1,1474 @@
+{
```

Comment:
<p>You don't need to do this now, but it could be good to look into the IPython magics for BQ to simplify some of the code. <a href="https://cloud.google.com/python/docs/reference/bigquery/latest/magics" target="_blank">https://cloud.google.com/python/docs/reference/bigquery/latest/magics</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/381/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>The issue with magics for BQ is that it doesn't support us to parameterize the dataset or project (e.g. in the FORM clause) which is a showstopper </p>